confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -185,20 +198,26 @@ private void updateLoadStatistics() {
     // Broadcast the RegionRouteMap if some LoadStatistics has changed
     boolean isNeedBroadcast = false;
 
-    // Update NodeStatistics
-    Map<Integer, NodeStatistics> differentNodeStatisticsMap = new ConcurrentHashMap<>();
+    // Update NodeStatistics:
+    // NodeStatistics[]:index 0 means the current NodeStatistics, index 1 means the previous
+    // NodeStatistics
+    Map<Integer, Pair<NodeStatistics, NodeStatistics>> differentNodeStatisticsMap =
+        new ConcurrentHashMap<>();
     getNodeManager()
         .getNodeCacheMap()
         .forEach(
             (nodeId, nodeCache) -> {
+              NodeStatistics preNodeStatistics = nodeCache.getPreviousStatistics().deepCopy();
               if (nodeCache.periodicUpdate()) {
                 // Update and record the changed NodeStatistics
-                differentNodeStatisticsMap.put(nodeId, nodeCache.getStatistics());
+                differentNodeStatisticsMap.put(
+                    nodeId, new Pair<>(nodeCache.getStatistics(), preNodeStatistics));
               }
             });
     if (!differentNodeStatisticsMap.isEmpty()) {
       isNeedBroadcast = true;
       recordNodeStatistics(differentNodeStatisticsMap);
+      eventBus.post(differentNodeStatisticsMap);
```

Comment:
If all the listeners want to interact with the target nodes, is there any way to allocate info and deliver once, just like DataNode start-up check.